spec/lib/percy/capybara/loaders/sprockets_loader_spec.rb

```diff
@@ -78,7 +78,7 @@ def self.call(_env)
       expect(resources.first.content).to include('.colored-by-base')
     end
     context 'Rails app' do
-      before(:each) do
```

Comment:
`before(:each)` not allowed?

Reply:
@djones before(:each) is allowed, but rubocop recommends against putting expectations in them.  So, I've setup `setup_rails_double` to create the double, and hold the shared expectations across the tests.